dockerplugin.py

```diff
@@ -198,11 +198,14 @@ def read_cpu_stats(container, dimensions, stats, t):
 
 def read_network_stats(container, dimensions, stats, t):
     """Process network utilization stats for a container."""
-    net_stats = stats['network']
+    net_stats = stats['networks']
```

Comment:
I'm assuming that's what's returned by the client library regardless of the version of the Docker daemon you talk to, as long as it uses API v1.21 or above?

Reply:
Yes, the client uses versioned endpoints to the docker daemon so the output is always in this format.